test/test_datapipe.py

```diff
@@ -2015,19 +2015,23 @@ class _CustomIterDataPipe(IterDataPipe):
         def noop(self, x):
             return x + 1
```

Comment:
Nit: maybe not call it "noop" if it does something.

Reply:
This has been fixed